app/js/components/manageApps/ManageApps.jsx

```diff
@@ -159,25 +159,10 @@ export default class ManageApps extends React.Component {
       const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
-        response.data.results.forEach(data => {
-          installedModules.push({
-            appDetails: data,
-            appType: "module",
-            install: false
-          });
-        });
-        installedAddons = installedOwas.concat(installedModules);
         return Promise
```

Comment:
Check out this [PR](https://github.com/openmrs/openmrs-owa-addonmanager/pull/109/files) and confirm that the code that you have removed is not needed

Reply:
Thank you, the PR has been updated